src/Microsoft.DotNet.Tools.Compiler/Program.cs

```diff
@@ -20,6 +20,9 @@ namespace Microsoft.DotNet.Tools.Compiler
 {
     public class Program
     {
+        private const string BuildProfile = "--build-profile";
```

Comment:
Why not follow the conventions of the other options and just put this in the app.option?